HEMCO/Extensions/hcox_seasalt_mod.F90

```diff
@@ -290,23 +383,170 @@ SUBROUTINE HCOX_SeaSalt_Run( am_I_Root, ExtState, HcoState, RC )
        SCALE = 0.329d0 + 0.0904d0*SST - &
                0.00717d0*SST**2d0 + 0.000207d0*SST**3d0
 
+       ! Limit the SST scaling factor to 0.25 over cold SST (below 5C)
+       IF ( Inst%ColdSST .and. SST<= 5.0d0 ) SCALE = 0.25d0
+
        ! Reset to using original Gong (2003) emissions (jaegle 6/30/11)
        !SCALE = 1.0d0
 
+       ! Apply to only the open ocean fraction of the gridbox (Huang 06/12/20)
+       FROPEN = ExtState%FROCEAN%Arr%Val(I,J)-ExtState%FRSEAICE%Arr%Val(I,J)
+
+       IF ( FROPEN < 0d0 ) FROPEN = 0d0
        ! Eventually apply wind scaling factor.
-       SCALE = SCALE * Inst%WindScale
+       SCALE = SCALE * Inst%WindScale * FROPEN
+
+       ! There is no need for this conditions as FROPEN=0 if there is
+       ! no open ocean. Also this lead to zeroing of SSA emissions over
+       ! polar night regions 
+       !inhibit emission if not ocean grid box
+       !IF ( HCO_LANDTYPE( ExtState%WLI%Arr%Val(I,J), &
+       !     ExtState%ALBD%Arr%Val(I,J) ) /= 0 ) &
+       !     SCALE = 0d0
```

Comment:
Since this is commented out does it need to be included?

Reply:
Yes, this can be removed. Note that using the following condition:
IF ( HCO_LANDTYPE( ExtState%WLI%Arr%Val(I,J), &
                                    ExtState%ALBD%Arr%Val(I,J) ) /= 0 )
to determine whether a grid box has open ocean, is present in other parts of HEMCO (such as hcox_seaflux_mod.F90 among others). The ALBD field in the MERRA-2 A1 file is set to a large number (>0.8) for all regions that are in polar night - I assume that this is because it is used/calculated in the radiation part of the MERRA-2 code. As a result, the HCO_LandType function (or any use of the condition ALBD>0.695 or ALBD>0.4) erroneously assumes that open ocean regions that are in polar night are covered by sea ice. Thus no sea salt emissions (or air-sea exchange) occurs in these boxes. This should be corrected. 

One simple fix is to replace the use of ALBD in HCO_LANDTYPE with sea ice fraction (FRSEAICE) and more generally to replace the use of ALBD with FRSEAICE to determine whether an ocean box is covered by sea ice or not.

Reply:
Thanks! Could you create a github issue with this bug report?